apps/remote_control/lib/lexical/remote_control/build/document/compilers/quoted.ex

```diff
@@ -0,0 +1,127 @@
+defmodule Lexical.RemoteControl.Build.Document.Compilers.Ast do
+  alias Elixir.Features
+  alias Lexical.Document
+  alias Lexical.RemoteControl.Build
+  alias Lexical.RemoteControl.ModuleMappings
+
+  import Lexical.RemoteControl.Build.CaptureIO, only: [capture_io: 2]
+
+  def compile_quoted(%Document{} = document, quoted_ast, compiler_name) do
```

Comment:
<img width="1573" alt="CleanShot 2023-07-29 at 19 32 58@2x" src="https://github.com/lexical-lsp/lexical/assets/12830256/f5ca80bf-830e-49c5-b4aa-898954093648">
<img width="1408" alt="CleanShot 2023-07-29 at 19 32 53@2x" src="https://github.com/lexical-lsp/lexical/assets/12830256/7b6eb83f-892d-44ed-b744-8d488a52b948">
Don't understand why duplicate these two files.

Reply:
It was a rename that went astray.